wp-content/themes/pub/wporg-learn-2020/css/customiser.css

```diff
@@ -0,0 +1,159 @@
+thead th {
+	color: #555d66;
+}
+
+.card,
+body:not(.single):not(.search) .site-main .card.post.course {
+	padding: 1.75rem;
+}
+
+/* Details summary block START */
+details {
+	border: 1px solid #e7e7e7;
+	padding: 0 .9em;
+	background: #f7f7f7;
+	margin: 1em 0;
+	display: flex;
+}
+
+details summary::before {
+	content: "\025B8";
+	margin-right: .5em;
+	font-size: 200%;
+	display: inline;
+	vertical-align: bottom;
+	line-height: 1.4;
+}
+
+details[open] summary::before {
+	content: "\025BE";
+}
+
+details summary {
+	display: inline-block;
+	font-weight: bold;
+	cursor: pointer;
+	line-height: 2.8;
+}
+
+/* Details summary block END */
```

Comment:
We're missing this block unless I'm mistaken?

Reply:
Yes, you were right. I missed that bit, I made a mess on my local and removed it and then I didn't add it back. Anyway, I added this in the override file. Let me know what you think.

Thanks.